com.unity.multiplayer.mlapi/Editor/MLAPIProfiler.cs

```diff
@@ -41,8 +40,8 @@ class ProfilerContainer
 
             public byte[] ToBytes()
 			{
-				BitStream stream = new BitStream();
-				BitWriter writer = new BitWriter(stream);
+				NetworkStream stream = new NetworkStream();
```

Comment:
This seems a bit off, because BitStream really isn't about networking.  The voting in the survey had keeping BitStream as the 'winner' - was there something else motivating this?

Reply:
oh, yes! we shouldn't rename `BitStream` as `NetworkStream` but we should call it `NetworkBuffer` instead, that's what we aligned on. there are several reasons why we picked "Network" prefix and replaced "Stream" with "Buffer" here. I will bring it up in our next stand-up. up until then, I'll commit changes to rename it as `NetworkBuffer` instead of `NetworkStream` (and `PooledNetworkBuffer`, `NetworkBufferPool` too).